metadata/nz_building_outlines_full_history.xml

```diff
@@ -104,50 +104,19 @@
         </gmd:CI_Citation>
       </gmd:citation>
       <gmd:abstract>
-        <gco:CharacterString>This dataset consists of building outlines within mainland New Zealand. This is the full history dataset containing all combinations of outline / building / name / use that have existed within the building outlines system, and the dates for which that combination existed. For a version of the data containing only the current building outlines see NZ Building Outlines. These datasets do not have full coverage of the country and will be added to as new imagery becomes available. 
-
-This dataset includes the spatial coverage of building outlines using remotely sensed information. A building outline is a 2D representation of the roof outline of buildings which have been classified from remotely sensed information using a combination of automated and manual processes to extract and orthogonalise building roof outlines. Structures greater than or equal to 10 square meters are captured in this dataset. Each building polygon represents a building outline and this may include spaces such as decks, garages and porches.
+        <gco:CharacterString>
 
-**Source imagery**
-Resolution of the imagery used for capture ranges from 50cm pixel resolution (0.5m GSD) to 7.5cm pixel resolution (0.075m GSD).  Please refer to the [NZ Building Outlines Data Dictionary](https://nz-building-outlines.readthedocs.io/en/latest/introduction.html#source-imagery) for further information.
+This dataset consists of polygon outlines representing buildings within mainland New Zealand. A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and orthogonalise a building roof outline. Building roof outlines observed in aerial imagery larger than or equal to 10 square meters are captured in this dataset. Each building polygon represents a building roof outline and this may include structures such as garages and large sheds.
 
-**Accuracy Specification:**
-The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery. The building outlines were created by an automated method of feature extraction. 
+This dataset contains the full history of all building outlines which have been extracted from multiple years of aerial imagery available in each region of the country, including the most recent imagery. This dataset includes the dates for which each building existed in the aerial imagery. Coverage areas include Northland, Auckland, Waikato, Hawke’s Bay, Manawatu-Whanganui, Wellington, Tasman, West Coast, Marlborough, Canterbury, Southland and Otago. As new aerial imagery becomes available, this dataset will be updated with new building outlines derived from this imagery.
```

Comment:
suggestions:

**Data vintage**
This dataset contains the full history of all building outlines which have been extracted from multiple years of aerial imagery available in each region of the country, including the most recent imagery. This dataset includes the dates for which each building existed in the aerial imagery.

Coverage areas include Northland, Auckland, Waikato, Hawke’s Bay, Manawatu-Whanganui, Wellington, Tasman, West Coast, Marlborough, Canterbury, Southland and Otago. This dataset will be updated and expanded as new aerial imagery becomes available.